docs/api/endorsement-provisioning/README.md

```diff
@@ -0,0 +1,104 @@
+# Endorsement Provisioning Interface
+
+The APIs described here are meant to illustrate the provisioning of 
+Endorsements based on RATS architecture using the Data Model as
+illustrated in 
+[Concise Reference Integrity Manifest](https://datatracker.ietf.org/doc/draft-birkholz-rats-corim/).
+
+## Provisioning API
+
+Provisioning API provides a facility to provision either Reference Values or Endorsed Values, using a 
+single CoRIM or a batch of CoRIM(s) in a single payload.
+
+* A client who intends to submit a CoRIM or a batch of CoRIM invokes a `POST` on a submit path;
+* If the transaction is complete then a `(200)`response is returned to the client. This is an indication
+  that the submitted CoRIM or batch has been successfully provisioned in a verifier. The ongoing transaction is now complete.
+* For large submission with multiple CoRIMs, the provisioning processing may happen in background. In this case,
+  Client is returned a `(202)` response, with a status of `processing`. Also 
```

Comment:
we said this was `201` with the Link header populated with the session resource?